dialogplus/src/main/java/com/orhanobut/dialogplus/Utils.java

```diff
@@ -60,15 +60,14 @@ static View getView(Context context, int resourceId, View view) {
    * @return the id of the animation resource
    */
   static int getAnimationResource(int gravity, boolean isInAnimation) {
-    switch (gravity) {
-      case Gravity.TOP:
-        return isInAnimation ? R.anim.slide_in_top : R.anim.slide_out_top;
-      case Gravity.BOTTOM:
-        return isInAnimation ? R.anim.slide_in_bottom : R.anim.slide_out_bottom;
-      case Gravity.CENTER:
-        return isInAnimation ? R.anim.fade_in_center : R.anim.fade_out_center;
-      default:
-        // This case is not implemented because we don't expect any other gravity at the moment
+    if ((gravity & Gravity.TOP) == Gravity.TOP) {
+      return isInAnimation ? R.anim.slide_in_top : R.anim.slide_out_top;
+    } else if ((gravity & Gravity.BOTTOM) == Gravity.BOTTOM) {
```

Comment:
since it's returned in each if-else block, I think we can simply avoid using else
```java
if ((gravity & Gravity.TOP) == Gravity.TOP) {
 return isInAnimation ? R.anim.slide_in_top : R.anim.slide_out_top 
}
if ((gravity & Gravity.BOTTOM) == Gravity.BOTTOM) {
  return ..
}
```